build.gradle

```diff
@@ -9,7 +9,7 @@ apply plugin: 'maven-publish'
 apply from: 'external-dependencies.gradle'
 apply from: 'idea-config.gradle'
 
-def majorMinorVersion = '0.1'
+def majorMinorVersion = '0.2'
```

Comment:
I also like the major.minor.patch pattern. I guess using the build number allows something similar to a SNAPSHOT build without the uncertainty that brings.